translations/harbour-seaprint-de.ts

```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
     <message>
         <source>Envelope Chou 2</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Chou 2</translation>
     </message>
     <message>
         <source>Envelope Chou 3</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Chou 3</translation>
     </message>
     <message>
         <source>Envelope Chou 40</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Chou 40</translation>
     </message>
     <message>
         <source>Envelope Chou 4</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Chou 4</translation>
     </message>
     <message>
         <source>Hagaki</source>
-        <translation type="unfinished"></translation>
+        <translation>Hagaki (Japanische Postkarte)</translation>
     </message>
     <message>
         <source>Envelope Kahu</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Kahu</translation>
     </message>
     <message>
         <source>270 x 382mm</source>
-        <translation type="unfinished"></translation>
+        <translation>270 x 382 mm</translation>
     </message>
     <message>
         <source>Envelope Kahu 2</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Kahu 2</translation>
     </message>
     <message>
         <source>216 x 277mm</source>
-        <translation type="unfinished"></translation>
+        <translation>216 x 277 mm</translation>
     </message>
     <message>
         <source>197 x 267mm</source>
-        <translation type="unfinished"></translation>
+        <translation>197 x 267 mm</translation>
     </message>
     <message>
         <source>190 x 240mm</source>
-        <translation type="unfinished"></translation>
+        <translation>190 x 240 mm</translation>
     </message>
     <message>
         <source>142 x 205mm</source>
-        <translation type="unfinished"></translation>
+        <translation>142 x 205 mm</translation>
     </message>
     <message>
         <source>119 x 197mm</source>
-        <translation type="unfinished"></translation>
+        <translation>119 x 197 mm</translation>
     </message>
     <message>
         <source>Oufuku Reply Postcard</source>
-        <translation type="unfinished"></translation>
+        <translation>Oufuku Reply (Japanische Postkarte)</translation>
     </message>
     <message>
         <source>Envelope You 4</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag You 4</translation>
     </message>
     <message>
         <source>10 x 11″</source>
-        <translation type="unfinished"></translation>
+        <translation>10 x 11″</translation>
     </message>
     <message>
-        <source>10 x 13″</source>
-        <translation type="unfinished"></translation>
+        <source>10 x 13 Zoll</source>
```

Comment:
Please don't break the source translation.

Reply:
</message>
    <message>
        <source>10 x 13 Zoll</source>
        <translation>10 x 13 Zoll</translation>
    </message>
    <message>
nothing is broken

Reply:
The translation is fine, but you changed the _source_.